docs/contributor_guide/contributing_example_notebooks.rst

```diff
@@ -17,16 +17,60 @@ A good example notebook exhibits the following attributes:
    Avoids abstraction traps.
    See `Selbst et al. (2020) <https://andrewselbst.files.wordpress.com/2019/10/selbst-et-al-fairness-and-abstraction-in-sociotechnical-systems.pdf>`_.
 4. **Substantiated**: Discusses trade-offs and compares alternatives.
-   Describes why using particular Fairlearn functionalities makes sense.
+   Describes why using particular Fairlearn functionalities makes sense if
+   Fairlearn is used.
 5. **For developers**: Speaks the language of developers and data scientists.
    Considers real practitioner needs.
-   Fits within the lifecycle of real practioner work.
+   Fits within the lifecycle of real practitioner work.
    See `Holstein et al (2019) <https://arxiv.org/pdf/1812.05239.pdf>`_,
    `Madaio et al. (2020) <http://www.jennwv.com/papers/checklists.pdf>`_.
 
 Please keep these in mind when creating, discussing, and critiquing examples.
 
-If you'd like to collaborate, please :ref:`reach out <communication>`.
+Process
+^^^^^^^
+
+All current efforts are tracked through items in the corresponding
+`Project Board <https://github.com/fairlearn/fairlearn/projects/3>`_.
+If you'd like to suggest a different kind of project, please
+`open a new issue <https://github.com/fairlearn/fairlearn/issues/new/choose>`_
+if you cannot find a similar one that is already tracked in the project board.
+If you'd like to collaborate with others from the community, please
+:ref:`reach out <communication>` to share your idea.
```

Comment:
Do you intend to have ALL issues on the project board? I don't think that'd be sustainable in the long run. Users can simply search in the issue tracker, the board is more like what we're focusing on that a given time?

Reply:
Hmm, so far that was indeed my plan. I guess 50 issues are a lot easier to handle than 1500 (scikit-learn). It's sort of nice to have similar types of tasks grouped together, which can be done with tags, too. We should discuss this further in #504 . I'm really open to it either way. I'll update this text to say "if you cannot find a similar one in the issue tracker".